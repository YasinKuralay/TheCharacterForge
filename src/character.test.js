import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import Character from "./character.js";

it("renders the character component without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Character />, div);
    ReactDOM.unmountComponentAtNode(div);
});

it("matches the previous snapshot of the Character Component", () => {
    const CharacterComponentSnapshot = renderer.create(<Character />).toJSON();
    expect(CharacterComponentSnapshot).toMatchSnapshot();
});
