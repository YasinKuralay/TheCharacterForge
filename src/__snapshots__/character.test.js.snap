// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the previous snapshot of the Character Component 1`] = `
<div>
  <div
    id="sidebar"
  >
    <div
      className="sidebarCard sidebarPersonName"
    >
      <i
        className="fas fa-portrait"
      />
       
      <input
        autoComplete="off"
        id="charNameField"
        onKeyDown={[Function]}
      />
    </div>
    <div
      id="addBar"
    >
      <i
        className="fas fa-folder-plus"
      />
      <i
        className="fas fa-plus-square"
        onClick={[Function]}
      />
    </div>
  </div>
</div>
`;
